js/map_utils.js

```diff
@@ -7,6 +7,12 @@ var zoom_scale;
 // Create the state polygons
 function addStates(map, stateBounds) {
 
+  // changes pointer events depending on mobile or desktop
+  var clickClass = "state-click-off"; // default to desktop
+  if(waterUseViz.mode === "mobile") {
+    clickClass = "state-click-on";
```

Comment:
i'm surprised that this issue required modifying state or county click behaviors at the national view level. isn't the perceived click/tap behavior in this view identical regardless of whether you're in desktop or mobile? in both cases, you click/tap within a state boundary and get zoomed in on that state. at least in desktop, we can hide from the user the implementation detail that it's actually a click on a county polygon, not a state polygon, that technically triggers the zoom.

on the other hand, maybe, this change in behavior could give us the option of not loading county boundaries at all until the national view is fully rendered and/or the user zooms in. but this PR doesn't take that shortcut, right? so if we want to do that, maybe we should introduce a new issue to optimize the loading?

Reply:
County clicks are different on mobile than desktop. On mobile, county clicks give you the water use value but on desktop you can hover and get that information. Therefore, to get the state-level water use data on mobile, you would need to be able to click to get that information but clicking a county from the national view would skip that piece.

Reply:
Created #188 to optimize mobile loading to take advantage of this PR.